src/app/user-profile/pages/edit-profile-page/edit-profile-page.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container edit-profile-container">
+  <mat-card class="edit-profile-card">
+    <mat-card-header>
+      <mat-card-title>{{'editProfile.edit-profile' | translate}}</mat-card-title>
+    </mat-card-header>
+    <form (ngSubmit)='submit()' [formGroup]="editProfileForm" class="edit-profile-form">
+      <mat-form-field appearance="fill">
+        <mat-label>{{'editProfile.name' | translate}}</mat-label>
+        <input type="text" matInput name="name" formControlName="name">
+        <!-- <mat-error *ngIf="hasNameError">{{ getNameErrorMessage() | translate }}</mat-error> -->
+      </mat-form-field>
+      <mat-form-field appearance="fill">
+        <mat-label>{{'editProfile.login' | translate}}</mat-label>
+        <input type="text" matInput name="login" formControlName="login">
+        <!-- <mat-error *ngIf="hasEmailError">{{ getEmailErrorMessage() | translate }}</mat-error> -->
+      </mat-form-field>
+      <mat-form-field appearance="fill">
+        <mat-label>{{'editProfile.password' | translate}}</mat-label>
+        <input type="password" matInput name="password" formControlName="password">
+        <!-- <mat-error *ngIf="hasPasswordError">{{ getPasswordErrorMessage() | translate }}</mat-error> -->
```

Comment:
Implement all error displaying methods that you commented out

Reply:
done